_component_design/cards.md

```diff
@@ -76,20 +50,11 @@ The card pattern is composed of the following three elements:
 <div class="hxRow"  markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Spacing
-
-Each section of a card uses medium container spacing, 20px, by default.
-
-Small and large spacing values can be used for card variations but should not be mixed and matched within a layout or card.
-
+## Specifications
+Use the following specifications when constructing cards.
 
 {% endcolumn %}
```

Comment:
Remove lines 56 to 65 if the specification block should be next to the image. If it should not be, then line 51 should use the col8 grid spacing.